Automattic/simplenote-android
# Issue/721 add open preview

### Fix
Add opening notes on the last-viewed tab (i.e. ***Edit*** or ***Preview***) to close #721.  The property is set on a per-note basis.  The option was originally added as a manual switch on the note info sheet, but the switch was removed in favor of automatically detecting the last-viewed tab to minimize the required interaction for the user.  Notes will open in the ***Edit*** tab by default unless the note is exited while the ***Preview*** tab is shown.  See the animation below for illustration.

<img src="https://user-images.githubusercontent.com/3827611/62569635-09072800-b84c-11e9-8e52-1c469a2058cb.gif" width="320">

### Test
1. Tap note in list with markdown on.
2. Notice ***Edit*** tab is shown.
3. Tap ***Preview*** tab.
4. Tap back arrow in app bar.
5. Tap note in list from Step 1.
6. Notice ***Preview*** tab is shown.
7. Tap ***Edit*** tab.
8. Tap back arrow in app bar.
9. Tap note in list from Step 1.
10. Notice ***Edit*** tab is shown.
11. Tap ***Info*** action in app bar.
12. Tap ***Enable markdown*** switch.
13. Notice ***Enable markdown*** switch is off.
14. Notice ***Edit*** and ***Preview*** tabs are not shown.
15. Tap back arrow in navigation bar.
16. Tap back arrow in app bar.
17. Tap note in list from Step 1.
18. Notice ***Edit*** and ***Preview*** tabs are not shown.
19. Notice note is in edit mode.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in b9689a2 with:
> Added showing the last viewed tab when opening a note
